src/index.ts

```diff
@@ -36,7 +35,7 @@ const plugin: Plugin<PluginOptions> = (editor, opt = {}) => {
     customComponents: [],
     importPlaceholder: '',
     imagePlaceholderSrc: '',
-    mjmlParser: mjml2html,
+    mjmlParser: opt.mjmlParser ? opt.mjmlParser : require("./components/parser"),
```

Comment:
any reason behind this? I think the require dependency will be "hoisted" anyway in the build